### PR TITLE
Update: Emulationstation.Emulationstation version 2.0.1a

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/20486)

### CODE REVIEW

**Comment by vedantmgoyal2009** [comment]
This is a different version. How can it be the same?

**Comment by quhxl** [comment]
I've updated the old manifest. Once the PR (#20487) is accepted, this should work.

**Comment by vedantmgoyal2009** [comment]
Thanks

File: manifests/e/Emulationstation/Emulationstation/2.0.1a/Emulationstation.Emulationstation.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+Installers:
+- InstallerLocale: en-US
+  Architecture: x86
+  InstallerType: nullsoft
+#  Scope:
```
Comment (ItzLevvie) [inline manifests/e/Emulationstation/Emulationstation/2.0.1a/Emulationstation.Emulationstation.installer.yaml:15]:
```suggestion
  Scope: machine
```

File: manifests/e/Emulationstation/Emulationstation/2.0.1a/Emulationstation.Emulationstation.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+PackageIdentifier: Emulationstation.Emulationstation
+PackageVersion: 2.0.1a
+PackageLocale: en-US
+Publisher: Emulationstation
+PublisherUrl: https://emulationstation.org/index.html
+PublisherSupportUrl: https://emulationstation.org/faq.html
+#PrivacyUrl: 
+#Author: 
+PackageName: Emulationstation
```
Comment (ItzLevvie) [inline manifests/e/Emulationstation/Emulationstation/2.0.1a/Emulationstation.Emulationstation.locale.en-US.yaml:12]:
```suggestion
PackageName: EmulationStation
```

File: manifests/e/Emulationstation/Emulationstation/2.0.1a/Emulationstation.Emulationstation.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,22 @@
+﻿# Created using YamlCreate.ps1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultlocale.1.0.0.schema.json
+
+PackageIdentifier: Emulationstation.Emulationstation
+PackageVersion: 2.0.1a
+PackageLocale: en-US
+Publisher: Emulationstation
```
Comment (ItzLevvie) [inline manifests/e/Emulationstation/Emulationstation/2.0.1a/Emulationstation.Emulationstation.locale.en-US.yaml:7]:
```suggestion
Publisher: EmulationStation
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/e/Emulationstation/Emulationstation/2.0.1a/Emulationstation.Emulationstation.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,21 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultlocale.1.0.0.schema.json
+
+PackageIdentifier: Emulationstation.Emulationstation
+PackageVersion: 2.0.1a
+PackageLocale: en-US
+Publisher: EmulationStation
+PublisherUrl: https://emulationstation.org/index.html
+PublisherSupportUrl: https://emulationstation.org/faq.html
+#PrivacyUrl: 
+#Author:
```
Comment (OfficialEsco) [inline manifests/e/Emulationstation/Emulationstation/2.0.1a/Emulationstation.Emulationstation.locale.en-US.yaml:10]:
```suggestion
PublisherSupportUrl: https://github.com/Aloshi/EmulationStation/issues
#PrivacyUrl: 
Author: Alec Lofquist
```

File: manifests/e/Emulationstation/Emulationstation/2.0.1a/Emulationstation.Emulationstation.locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+#Author: 
+PackageName: EmulationStation
+PackageUrl: https://emulationstation.org/index.html#download
+License: Copyright (c) 2014 Alec Lofquist
+LicenseUrl: https://github.com/Aloshi/EmulationStation/blob/master/LICENSE.md
+#Copyright: 
+#CopyrightUrl: 
+ShortDescription: A graphical and themeable emulator front-end that allows you to access all your favorite games in one place, even without a keyboard!
+#Description: 
+Moniker: emulationstation
```
Comment (OfficialEsco) [inline manifests/e/Emulationstation/Emulationstation/2.0.1a/Emulationstation.Emulationstation.locale.en-US.yaml:19]:
```suggestion
Moniker: emulationstation
Tags:
- cross-platform
- emulator
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**